eos-tech-support/eos-stage-flatpak

```diff
@@ -58,6 +85,10 @@ else
 fi
 
 for remote in `flatpak remote-list --system | awk {'print $1'}` ; do
+    if [ -n "$REMOTE_NAME" ] && [ "$remote" != "$REMOTE_NAME" ]; then
+        continue
+    fi
```

Comment:
Actually, one other comment: some error handling should be added to warn the user if they specify a `--remote-name` which doesn’t actually exist (i.e. if they typo it), because otherwise this script will trivially succeed.